src/librustc/mir/interpret/value.rs

```diff
@@ -158,32 +173,32 @@ impl<'tcx, Tag> Scalar<Tag> {
     }
 
     #[inline]
-    pub fn ptr_offset(self, i: Size, cx: impl HasDataLayout) -> EvalResult<'tcx, Self> {
+    pub fn ptr_wrapping_offset(self, i: Size, cx: impl HasDataLayout) -> Self {
```

Comment:
looks like you swapped this function and the one below? I think the diff is cleaner the other way around